modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -102,6 +102,8 @@ As an alternative to `.spec.resourceCustomizations`, you can use `.spec.resource
 To prevent data loss during upgrade to {gitops-title} Operator v1.10.0, ensure that you backup `.spec.resourceCustomization` value if it is used in your Argo CD CRs.
 ==== 
 
+* With this update, support for deprecated Legacy Configuration Management Plugins (CMPs) specified in `argo-cm` or through the Operator `argocd.spec.configManagementPlugins` is removed in Argo CD v2.8. To continue using your legacy plugins, consider migrating them to the new sidecar available in the Operator through `argocd.spec.repo.sidecarContainers`. link:https://issues.redhat.com/browse/GITOPS-3462[GITOPS-3462]
```

Comment:
I am not sure that the word "Operator" is accurate here as the `configManagementPlugins` object is described as Argo CD cm property here in this table: [Argo CD custom resource properties](https://docs.openshift.com/gitops/1.10/argocd_instance/argo-cd-cr-component-properties.html#argo-cd-properties_argo-cd-cr-component-properties). But I am not sure. Could you please check?

```suggestion
* With this update, support for deprecated Legacy Configuration Management Plugins (CMPs) specified in `argo-cm` or through the `argocd.spec.configManagementPlugins` parameter is removed in Argo CD v2.8. To continue using your legacy plugins, consider migrating them to the new sidecar available in the Operator through `argocd.spec.repo.sidecarContainers`. link:https://issues.redhat.com/browse/GITOPS-3462[GITOPS-3462]
```

Reply:
@svghadi here too, need clarity

Reply:
The "**\<type-of-object\>**" here is "**field in Argo CD CR**".